code/datums/diseases/transformation.dm

```diff
@@ -128,7 +128,7 @@
 		add_monkey(affected_mob.mind)
 	if(ishuman(affected_mob))
 		if(affected_mob && !is_monkey_leader(affected_mob.mind))
-			var/mob/living/carbon/monkey/M = affected_mob.monkeyize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPDAMAGE | TR_KEEPVIRUS | TR_KEEPSE)
+			var/mob/living/carbon/monkey/M = affected_mob.monkeyize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPVIRUS | TR_KEEPSE)
```

Comment:
Why is this edit here?

Reply:
Hm that shouldn't be there. Was messing with momke and must've saved by mistake.